elastic/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # CHANGELOG - elastic
 
+1.0.1 / 2017-07-21
```

Comment:
@greentruff could you please change this to `Unreleased`. We'll make sure we get this updated once we actually relase the check/agent. Thanks!!